test-suite/src/function/custom.rs

```diff
@@ -0,0 +1,127 @@
+use {
```

Comment:
Could we move this `function/custom.rs` to the root with a name of `custom_function.rs`?
Because unlike other normal functions, this is special case even with option store trait, so it would be good to emphasis more.